cmd/analyze/main.go

```diff
@@ -29,7 +29,7 @@ var (
 	staticUpload        = flag.String("upload-static", "", "bucket path for uploading static analysis results")
 	uploadFileWriteInfo = flag.String("upload-file-write-info", "", "bucket path for uploading information from file writes")
 	offline             = flag.Bool("offline", false, "disables sandbox network access")
-	combinedSandbox     = flag.Bool("combined-sandbox", false, "use combined dynamic analysis sandbox")
+	noCombinedSandbox   = flag.Bool("no-combined-sandbox", false, "don't use new combined dynamic analysis sandbox image")
```

Comment:
Rather than the negative naming for the boolean ("no ....") try renaming it in a way that doesn't require it (e.g. `separate-sandboxes`). This makes conditions easier to read.

For example: consider `notCold` versus `hot` (totally contrived). Then:
```
if !notCold {
  // this is confusing
}
// VERSUS
if !hot {
  // this is much clearer
}
```

Reply:
Done.

For what it's worth, `--no-xxx` options are quite common in CLI tools, and it makes sense when `xxx` is the default. For example, cURL has `--no-alpn`, `--no-buffer`, `--no-clobber`, `--no-keepalive`, `--no-npn`, `--no-progress-meter`, `--no-session-id`, `--noproxy`.